### PR TITLE
[FABC-800] Enlarge PEM column for mysql db from 4096 to 8192

### PR DESCRIPTION
**Pull request by caod123**
#### Type of change

- Bug fix

#### Description

Also update migrator

#### Related issues

[FABC-800](https://jira.hyperledger.org/browse/FABC-800)

/cc @adecaro

### CODE REVIEW

**Comment by caod123** [comment]
Not sure if this also needs to be updated in the migrator

File: scripts/fvt/dbmigration_test.sh [hunk 1]
```diff
@@ -119,7 +119,7 @@ mysql --host=localhost --user=root --password=mysql -e "drop database $DBNAME;"
...
 mysql --host=localhost --user=root --password=mysql --database=$DBNAME -e "CREATE TABLE affiliations (name VARCHAR(64) NOT NULL UNIQUE, prekey VARCHAR(64));"
-mysql --host=localhost --user=root --password=mysql --database=$DBNAME -e "CREATE TABLE certificates (id VARCHAR(64), serial_number varbinary(128) NOT NULL, authority_key_identifier varbinary(128) NOT NULL, ca_label varbinary(128), status varbinary(128) NOT NULL, reason int, expiry timestamp DEFAULT 0, revoked_at timestamp DEFAULT 0, pem varbinary(4096) NOT NULL, PRIMARY KEY(serial_number, authority_key_identifier)) DEFAULT CHARSET=utf8 COLLATE utf8_bin;"
+mysql --host=localhost --user=root --password=mysql --database=$DBNAME -e "CREATE TABLE certificates (id VARCHAR(64), serial_number varbinary(128) NOT NULL, authority_key_identifier varbinary(128) NOT NULL, ca_label varbinary(128), status varbinary(128) NOT NULL, reason int, expiry timestamp DEFAULT 0, revoked_at timestamp DEFAULT 0, pem varbinary(8192) NOT NULL, PRIMARY KEY(serial_number, authority_key_identifier)) DEFAULT CHARSET=utf8 COLLATE utf8_bin;"
```
Comment (sykesm) [inline scripts/fvt/dbmigration_test.sh:122]: IIUC, This is supposed to be the pre-migration table. The code in `lib/server/db/mysql/migrator.go` needs to changed to expand the column.

Validation that the column was expanded is lower down.

Comment (caod123) [reply scripts/fvt/dbmigration_test.sh]: Yep just realized that as I was updating the migrator

File: lib/server/db/mysql/migrator.go [hunk 2]
```diff
@@ -103,6 +103,10 @@ func (m *Migrator) MigrateCertificatesTable() error {
 		if err != nil {
 			return err
 		}
+		_, err = tx.Exec(funcName, "ALTER TABLE certificates MODIFY pem varbinary(8192)")
+		if err != nil {
+			return err
+		}
```
Comment (sykesm) [inline lib/server/db/mysql/migrator.go:109]: And this does not look like it would apply if the first migration took place, right?

Comment (caod123) [reply lib/server/db/mysql/migrator.go]: good catch

**Pull request merged**